src/mudlet-lua/lua/generic-mapper/generic_mapper.xml

```diff
@@ -2920,6 +2921,20 @@ function doSpeedWalk()
     end
 end
 
+function map.pauseSpeedwalk()
+    -- map.echo("pauseSpeedwalk!")
+    walking = false
+end
+
+function map.resumeSpeedwalk()
+    -- map.echo("resumeSpeedwalk!")
+    map.find_me(nil, nil, nil, true)
+    if #speedWalkDir ~= 0 then
+        walking = true
+        tempTimer(1, "doSpeedWalk()")
```

Comment:
Could a delay of `0` also work here?

Reply:
l test in mud, Will go wrong.  now I modify this function like this `map.resumeSpeedwalk(delay)` to set delay